### PR TITLE
RF-01.004.04 | FreestyleProject > Delete Project|Delete a project from the Project Page

### PR DESCRIPTION
**Pull request by KateVernikovskaya**
POM -  https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/503
Converted TC_01.004.04 to POM format

TC - https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/202
US - https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/22

### CODE REVIEW

File: cypress/e2e/freestyleDeleteProjectFromProjectPagePOM.cy.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import configurePageData from "../fixtures/configurePageData.json"
+
+const dashboardPage = new DashboardPage();
+const newJobPage = new NewJobPage();
+const freestyleProjectPage = new FreestyleProjectPage();
+
+
+describe('US_01.004 | FreestyleProject > Delete Project', () => {
+
+    it.only('TC_01.004.04 | FreestyleProject > Delete Project|Delete a project from the Project Page', () => {
```
Comment (Negrescu29) [inline cypress/e2e/freestyleDeleteProjectFromProjectPagePOM.cy.js:16]: Why this TC has .only?

File: cypress/pageObjects/FreestyleProjectPage.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ class FreestyleProjectPage {
     getDashboardLink = () => cy.get('a[href="/"].model-link');
     getDeleteMenuItem = () => cy.get('a[data-title="Delete Project"]');
     getCancelButton = () => cy.get('button[data-id="cancel"]');
+    getYesButton = () => cy.get('button[data-id="ok"]');
```
Comment (Negrescu29) [inline cypress/pageObjects/FreestyleProjectPage.js:20]: Just a suggestion (not sure how the UI looks like) to change the naming to something like getOKButton

**Review by Negrescu29 — APPROVED** [review]

File: cypress/e2e/freestyleDeleteProjectFromProjectPagePOM.cy.js (hunk 1 above)
Comment (KateVernikovskaya) [reply cypress/e2e/freestyleDeleteProjectFromProjectPagePOM.cy.js]: Да, спасибо ) исправляю

File: cypress/pageObjects/FreestyleProjectPage.js (hunk 2 above)
Comment (KateVernikovskaya) [reply cypress/pageObjects/FreestyleProjectPage.js:20]: Спасибо)выглядит так как написала, оставлю так)

**Review by Negrescu29 — APPROVED** [review]

**Review by ivanlashkevich — APPROVED** [review]
Looks good. Approved.

**Pull request merged**